examples/2d/pixel_grid_snap.rs

```diff
@@ -176,6 +177,6 @@ fn fit_canvas(
         let h_scale = event.width / RES_WIDTH as f32;
         let v_scale = event.height / RES_HEIGHT as f32;
         let mut projection = projections.single_mut();
-        projection.scale = 1. / h_scale.min(v_scale).round();
+        projection.scaling_mode = ScalingMode::WindowSize(h_scale.min(v_scale).round());
```

Comment:
Note: this appears to produce the same behaviour as the existing example, though if there's a simpler method I haven't seen that's great too!